azure/bastion/variables.tf

```diff
@@ -50,12 +50,17 @@ variable "whitelist" {
   description = "A list of allowed CIDR ranges or service tags to give access to the bastion. Default to Internet service tag."
   type        = list(string)
   default     = ["Internet"]
+
+  validation {
+    error_message = "Whitelist can not be an empty list."
+    condition     = length(var.whitelist) != 0
```

Comment:
I'd change this to:

`length(var.whitelist) >= 1`

Because != 0 also allows 

`whitelist = null`

which is not valid

Reply:
Makes sense - Changed